common/journal_entries/dvg_lotharingia.txt

```diff
@@ -0,0 +1,337 @@
+je_lotharian_house_of_cards = {
+	icon = "gfx/interface/icons/event_icons/event_scales.dds"
+
+	is_shown_when_inactive = {

```

Comment:
is_shown_when_inactive and possible can be removed if it should appear on startup event.
This is prob now blocking it from showing up on the event.